src/System.Net.Http/src/netcore50/project.json

```diff
@@ -16,6 +16,7 @@
         "System.Resources.ResourceManager": "4.0.0",
         "System.Runtime.Extensions": "4.0.10",
         "System.Runtime.WindowsRuntime": "4.0.10",
+        "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
@davidsh As long as we're not hoping to rush marking 4.1 as stable I'm good with "work out the problems as we go along".  The crypto packages are going to be stable by RTM, and that includes UWP unless we do a huge amount of gymnastics.